[BUG] Building error in Termux

Moving step-by-step in installation procedure, encountered a 3 critical error, build interrupted:
 ` Compiling purrcrypt v0.1.0 (/data/data/com.termux/files/home/purrcrypt)
error[E0433]: failed to resolve: could not find `os` in `fs`
  --> src/keys.rs:18:9
   |
18 | use fs::os::unix::fs::PermissionsExt;
   |         ^^ could not find `os` in `fs`

error[E0599]: no function or associated item named `from_mode` found for struct `Permissions` in the current scope
  --> src/keys.rs:56:56
   |
56 |         fs::set_permissions(pub_path, fs::Permissions::from_mode(0o644))?;
   |                                                        ^^^^^^^^^ function or associated item not found in `Permissions`
   |
   = help: items from traits can only be used if the trait is in scope
help: trait `PermissionsExt` which provides `from_mode` is implemented but not in scope; perhaps you want to import it
   |
2  + use std::os::unix::fs::PermissionsExt;
   |

error[E0599]: no function or associated item named `from_mode` found for struct `Permissions` in the current scope
  --> src/keys.rs:71:59
   |
71 |         fs::set_permissions(secret_path, fs::Permissions::from_mode(0o600))?;
   |                                                           ^^^^^^^^^ function or associated item not found in `Permissions`
   |
   = help: items from traits can only be used if the trait is in scope
help: trait `PermissionsExt` which provides `from_mode` is implemented but not in scope; perhaps you want to import it
   |
2  + use std::os::unix::fs::PermissionsExt;
   |

Some errors have detailed explanations: E0433, E0599.
For more information about an error, try `rustc --explain E0433`.
error: could not compile `purrcrypt` (lib) due to 3 previous errors
error: failed to compile `purrcrypt v0.1.0 (/data/data/com.termux/files/home/purrcrypt)`, intermediate artifacts can be found at `/data/data/com.termux/files/home/purrcrypt/target`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path`

--- a/src/keys.rs
+++ b/src/keys.rs
@@ -15,7 +15,7 @@ use std::fs;
 use thiserror::Error;
 
 #[cfg(unix)]
-use fs::os::unix::fs::PermissionsExt;
+use std::os::unix::fs::PermissionsExt;
 
 #[derive(Error, Debug)]
 pub enum KeyError {

